config/routes.rb

```diff
@@ -1,18 +1,25 @@
+require 'sidekiq/web'
```

Comment:
Add documentation to the README about how to run sidekiq etc, in order to help your peers to run it as well